test/verify/check-machines

```diff
@@ -269,5 +269,38 @@ class TestMachines(MachineCase):
             b.wait(lambda: b.eval_js("document.documentElement.outerHTML.indexOf('noVNC_status_normal') >= 0"))
         b.wait(lambda: b.eval_js("document.documentElement.outerHTML.indexOf('Connected (unencrypted) to: QEMU (subVmTest1)') >= 0"))
 
+    def testDelete(self):
+        b = self.browser
+        m = self.machine
+
+        name = "subVmTest1"
+        img1 = "/var/lib/libvirt/images/{0}.img".format(name)
+        img2 = "/var/lib/libvirt/images/{0}-2.img".format(name)
+
+        self.startVm(name, console='vnc')
```

Comment:
Not that it matters much, but why is specifying "vnc" explicitly important here?